op-conductor/consensus/raft.go

```diff
@@ -110,11 +141,19 @@ func NewRaftConsensus(log log.Logger, cfg *RaftConsensusConfig) (*RaftConsensus,
 	// If bootstrap = true, start raft in bootstrap mode, this will allow the current node to elect itself as leader when there's no other participants
 	// and allow other nodes to join the cluster.
 	if cfg.Bootstrap {
+		var advertisedAddr raft.ServerAddress
+		if cfg.AdvertisedAddr == "" {
+			advertisedAddr = transport.LocalAddr()
```

Comment:
Shouldn't this be `advertisedAddr = bindAddr` ? I think this is the only behaviour change that might not be expected.

Reply:
If it binds to port 0, then the advertised address would contain a 0. The `transport.LocalAddr()` will return the resolved address after binding, which should always be safe to use I think.